benchmark/tpcds_plan_cost/queries/04.sql

```diff
@@ -0,0 +1,120 @@
+WITH year_total AS
```

Comment:
Are these intended to be added?

Reply:
no, must added it accidentally. will revert it